_posts/en/newsletters/2022-02-02-newsletter.md

```diff
@@ -0,0 +1,137 @@
+  and hashed, with the hash digest being put on the evaluation stack for
+  later opcodes to use.  A new [OP_CHECKSIGFROMSTACK][topic
+  op_checksigfromstack] (CSFS) opcode (as previously proposed) would
+  allow a tapscript to specify a public key and require a corresponding
+  signature over particular data on the stack---such as the computed
+  digest of the transaction created by `OP_TXHASH`.
+
+    O'Connor explained how these two opcodes would allow emulation of
+    two earlier soft fork proposals, [SIGHASH_ANYPREVOUT][topic
+    sighash_anyprevout] (APO, specified in [BIP118][]) and
+    [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] (CTV
+    specified in [BIP119][]).  For some purposes, the emulation might be
+    less efficient than using CTV or APO directly, but `OP_TXHASH` and
+    `OP_CSFS` would keep the Tapscript language simpler and provide more
+    flexibility for future script writers, especially if it was combined
+    with other simple additions to tapscript such as [OP_CAT][].
+
+    In a [reply][towns pop_sigdata], Anthony Towns suggested a similar
+    approach using other alternative opcodes.
+
+    The ideas were still being actively discussed as this summary was
+    being written.  We expect to revisit the topic in a future
+    newsletter.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BTCPay Server 1.4.2][] is the latest release in the new 1.4.x series,
+  which includes improvements in login authentication and a number of
+  [user interface improvements][btcpay ui blog].
+
+- [BDK 0.16.0][] is a release that includes a number of bug fixes and
+  small improvements.
+
+- [LND 0.14.2-beta.rc1][lnd 0.14.2-beta] is the release candidate for a
+  maintenance version that includes several bug fixes and a few minor
+  improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23201][] improves the ability of wallet users to fund
+  transactions with external inputs (previously mentioned in [Newsletter
+  #170][news170 external inputs]) by allowing them to specify maximum
+  weights instead of solving data.  This enables applications to use
+  `fundrawtransaction`, `send`, and `walletfundpsbt` RPCs to fee bump
+  transactions with nonstandard outputs such as HTLCs, a requirement for
+  Lightning clients described in [Newsletter #184][news184 eclair auto bump]).
```

Comment:
@harding I felt that linking to the summary of #17211 would be more relevant here, but feel free to remove.